controller/userController.js

```diff
@@ -0,0 +1,61 @@
+const jwt = require('jsonwebtoken');
+
+const User = require('../model/userSchema');
+
+exports.login = async (req, res) => {
+    const username = req.body.username;
+    const password = req.body.password;
+    try {
+        const user = await User.findOne({ username: username });
+        if (!user) {
+            res.status(401).json({ err: "A user with this email could not be found." });
+        }
+        if (user.password !== password)
+            res.status(401).json('Wrong password')
+        const token = jwt.sign(
+            {
+                username: user.username,
+                userId: user._id.toString()
+            },
+            'somereallylongandunpredictabletext',
+            { expiresIn: 120 }
```

Comment:
120 also can be read from .env file - you need to have a separate environment variable in .env file and read it's value here as follows: 

            { expiresIn: process.env.EXPIRES_IN }